src/plotting/plotSettings/moreOptions/moreOptionsBox.component.tsx

```diff
@@ -143,6 +169,53 @@ const MoreOptionsBox = (props: MoreOptionsProps) => {
             changeColour={changeChannelColour}
           />
         </Box>
+        <Grid container item>
+          <FormControl
+            sx={{
+              display: 'flex',
+              flexDirection: 'row',
+              width: 'inherit',
+              justifyContent: 'space-between',
+              border: 1,
+              padding: 1,
+              alignItems: 'center',
+            }}
+          >
+            <FormLabel
+              sx={{ fontSize: 12, color: 'inherit' }}
+              id="y-axis-label"
+            >
+              Y Axis
+            </FormLabel>
+            <RadioGroup
+              row
+              aria-labelledby="y-axis-label"
+              name="y-axis-label-radio-buttons"
+              value={thisChannel.options.yAxis}
+              onChange={(event) => {
+                const newValue = event.target.value;
+                changeChannelAxis(
+                  newValue as SelectedPlotChannel['options']['yAxis']
+                );
+              }}
+            >
+              <FormControlLabel
+                value="left"
+                control={<Radio size="small" sx={{ padding: '2px' }} />}
+                label="Left"
+                componentsProps={{ typography: { sx: { fontSize: 12 } } }}
+                sx={{ margin: 0 }}
+              />
+              <FormControlLabel
+                value="right"
+                control={<Radio size="small" sx={{ padding: '2px' }} />}
+                label="Right"
+                componentsProps={{ typography: { sx: { fontSize: 12 } } }}
+                sx={{ margin: 0 }}
+              />
```

Comment:
Could we add aria labels for these?

Reply:
These should all be accessible due to using controlled by and the fact that radios are labelled by text

Reply:
Inspecting accessibility properties on FF shows that all the roles and labels look fine & associated correctly

![image](https://user-images.githubusercontent.com/22525471/195104280-7140bbe8-75bb-4ace-a5aa-4017565c3885.png)